applications/nrf5340_audio/src/bluetooth/bt_content_control/bt_content_ctrl.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BT_CONTENT_CTRL_H_
+#define _BT_CONTENT_CTRL_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+/**
+ * @brief	Start content.
+ *
+ * @param	conn	Pointer to the connection to control, can be NULL.
```

Comment:
I was avoiding using this actually, since they usually don't give that much benefit and can easily be wrong.  

I see that in our project we are not consistent in the use of [in], [in, out] and [out]. Some cases are just missing them, some are plain wrong and some are partially wrong (saying that a parameter is [in, out] when it is in fact just [out]. So we should probably do an overhaul of all files at some point to get them all on the same standard.

Reply:
I see. Could you create an issue for this, given that it is maintenance of the code?

Reply:
I've also added [in] [out] [in, out] for all the header files in this PR